Changes Default Property Values

thaiguer opened the pull request:
# Description
This PR changes the Header Value for 3 values that were beeing incorrectly interpreted by BricsCad.

The image shows the error list from BricsCad prompt before and after the code changes.
<img width="2902" height="1720" alt="image" src="https://github.com/user-attachments/assets/7c5c348c-e76b-46ac-80bf-d479971a67a2" />

# Related Issues / Pull Requests
- This PR solves partly the problem being investigated on the issue #834

DomCR commented:
docs:
https://static.bricsys.com/help/Offline%20help/V21/en-US/2_system_variable_reference.pdf

thaiguer commented:
The remaing erros are just these 5:
```
: _RECOVER
Recover .dwg file.
Name: .dwg file
Value: Object improperly read: <AcDbRasterImage> (3D)
 Previous error: .dwg file needs recovery
Replaced by: Removed

Total errors found during recover: 1.
Audit recovered Database.
Name: Database header
Value: System Variable "CMLSTYLE"
Validation: Invalid
Replaced by: Standard

Name: Database header
Value: System Variable "TILEMODELIGHTSYNCH"
Validation: Not 0..1
Replaced by: Set to 1

Name: AcDbTextStyleTableRecord(12)
Value: Is not externally dependent on an Xref, but XRefBlockId ((4)) is not Null
Validation: Invalid
Replaced by: Set to Null

Name: AcDbLayout(35)
Value: Layout has invalid ModelType flag
Validation: Invalid
Replaced by: Set to True

67 objects audited
Total errors found during audit 5, fixed 5
```

DomCR commented:
I've been trying to locate the variable "TILEMODELIGHTSYNCH" but I couldn't find it, there are some sections in the dwg file that are not fully read because they are not rellevant for the file at all, al least for `ACadSharp`.

I'll try to focus on the main problem, the Value: Object improperly read: <AcDbRasterImage> (3D), I'm guessing that is because the `ImageRecord` is not written but I'm not sure. 

For the other issue "Value: Is not externally dependent on an Xref, but XRefBlockId ((4)) is not Null" not sure why is complaining about this one for a text style.

thaiguer commented:
> I've noticed that ClipType is set to Rectangular, which takes only the 2 first vertices to draw a box, try to change the vertices to:
> 
> ```cs
> raster.ClipBoundaryVertices.Add(new XY(0, 0));
> raster.ClipBoundaryVertices.Add(new XY(1, 1));
> ```
> 
> Or change the ClipType to polygonal, see if this does something.

```
using ACadSharp;
using ACadSharp.Entities;
using ACadSharp.IO;
using ACadSharp.Objects;
using CSMath;

string newFilePath = @"C:\devAux\newDocWithImage.dwg";
string imagePath = @"C:\devAux\image.jpg";

ImageDefinition definition = new ImageDefinition();
definition.Size = new XY(1, 1);
definition.Name = "image";
definition.IsLoaded = true;
definition.FileName = imagePath;

RasterImage raster = new RasterImage(definition);
raster.Flags = ImageDisplayFlags.ShowImage;

raster.ClipBoundaryVertices.Add(new XY(0, 0));
raster.ClipBoundaryVertices.Add(new XY(1, 1));
raster.ClipType = ClipType.Polygonal;

CadDocument newDoc = new CadDocument();
newDoc.Entities.Add(raster);

using (DwgWriter writer = new DwgWriter(newFilePath, newDoc))
{
    writer.Write();
}
```

```
: RECOVER
Recover .dwg file.
Name: .dwg file
Value: Object improperly read: <AcDbRasterImage> (3D)
 Previous error: .dwg file needs recovery
Replaced by: Removed

Total errors found during recover: 1.
Audit recovered Database.
Name: Database header
Value: System Variable "TILEMODELIGHTSYNCH"
Validation: Not 0..1
Replaced by: Set to 1

Name: AcDbTextStyleTableRecord(12)
Value: Is externally dependent on an Xref, but Xref BlockRecord ((4))
Validation: Invalid
Replaced by: Removed dependence

67 objects audited
Total errors found during audit 3, fixed 3
```
